### ISSUE TITLE
Describe what we expect and not expect from an architect

### ISSUE THREAD

**Issue by jelisejev** [ctco-dev/architect-requirements#17]
I think that before we start listing what is mandatory and not mandatory for an architect, it would be great to define, what we actually expect from an architect. Mainly, what are his day to day duties. We should also clearly state things we don't expect from an architect (such as deep technical understanding of each platform out there)

**Comment by nikita-golovin** [comment]
Duplicate of #15
